jest.config.js

```diff
@@ -28,7 +28,9 @@ module.exports = {
     'src/common/RootProviders',
     'src/common/LoggedInProviders',
     'src/common/ThemedNavigationContainer',
-    'src/navigators', 'src/components/TrackTile'
+    'src/navigators', 'src/components/TrackTile',
+    'src/screens',
```

Comment:
Why are `screens` being added?  Feel free to drop coverage thresholds instead.  The tricky thing about this exclusion list is that it's easy to forget about; it'd be good to keep screens revealing it's coverage

Reply:
I didn't exclude `src/screens` temporarily.  I don't think there is generally much logic in screens and therefore don't require tests.  My thought was that this coverage metric could be more accurate if it reflected only areas that generally give good ROI with testing.

Happy to add it back however.